tests/ui/privacy/sealed-traits/private-trait-non-local.stderr

```diff
@@ -0,0 +1,9 @@
+error[E0603]: module `index` is private
+  --> $DIR/private-trait-non-local.rs:2:18
+   |
+LL | use core::slice::index::private_slice_index::Sealed;
```

Comment:
I quite like this framing, though I wonder if we should either point to somewhere with privacy rules, or at least drive this home by saying something like "an item is only reachable via a path if all of the segments are reachable". Maybe better wording, but something that folks can take away from the error message as intuition for why public-unreachable messages happen the way they do?

Reply:
I'd like to add some note about it, but I'm not sure what it should look like at the moment.